### PR TITLE
Update icon for fido2 mds 

### PR DESCRIPTION
**Pull request by daringer**
* update icon to be a svg
* re-create all mds jsons `v5`

### CODE REVIEW

File: utils/fido2-mds/generate-mds.py [hunk 1]
```diff
@@ -85,7 +88,7 @@ def mds(self, ctap: CtapVersion) -> MetadataStatement:
             options["pinUvAuthToken"] = True
             pin_protocols = [2, 1]
         else:
-            options["credentialMgmtPreview"] = True,
+            options["credentialMgmtPreview"] = (True,)
```
Comment (sosthene-nitrokey) [inline utils/fido2-mds/generate-mds.py:91]: Why the one-element tuple?

**Review by sosthene-nitrokey — APPROVED** [review]

File: utils/fido2-mds/generate-mds.py [hunk 2]
```diff
@@ -59,9 +63,8 @@ def mds(self, ctap: CtapVersion) -> MetadataStatement:
             attestation_root_certificate = f.read()
 
         with open("nitrokey.svg", "rb") as f:
-            icon_bytes = svg2png(file_obj=f, output_width=128, output_height=128)
-            icon_base64 = base64.b64encode(icon_bytes).decode()
-            icon = f"data:image/png;base64,{icon_base64}"
+            icon_base64 = base64.b64encode(f.read()).decode()
+            icon = f"data:image/svg+xml;base64,{icon_base64}"
```
Comment (robin-nitrokey) [inline utils/fido2-mds/generate-mds.py:67]: The spec says:

> icon, of type [DOMString](https://heycam.github.io/webidl/#idl-DOMString)

>    A data: url [[RFC2397]](https://fidoalliance.org/specs/mds/fido-metadata-statement-v3.0-ps-20210518.html#biblio-rfc2397) encoded [[PNG]](https://fidoalliance.org/specs/mds/fido-metadata-statement-v3.0-ps-20210518.html#biblio-png) icon for the Authenticator.

https://fidoalliance.org/specs/mds/fido-metadata-statement-v3.0-ps-20210518.html#dom-metadatastatement-icon

**Review by robin-nitrokey — CHANGES_REQUESTED** [review]

File: utils/fido2-mds/generate-mds.py (hunk 1 above)
Comment (daringer) [reply utils/fido2-mds/generate-mds.py:91]: uh good catch, accidental `,` added

File: utils/fido2-mds/generate-mds.py (hunk 2 above)
Comment (daringer) [reply utils/fido2-mds/generate-mds.py:67]: uh, ok, the mymds portal seems to be just fine with the svg, but spec-conformance is for sure better

File: utils/fido2-mds/generate-mds.py (hunk 1 above)
Comment (daringer) [reply utils/fido2-mds/generate-mds.py]: done

**Review by robin-nitrokey — APPROVED** [review]

**Pull request merged**